src/histolab/tiler.py

```diff
@@ -151,6 +158,14 @@ def _tile_filename(
 
         return tile_filename
 
+    @lazyproperty
+    def _tiles_locator(self):
+        return (
+            self._highest_score_tiles
+            if isinstance(self, ScoreTiler)
+            else self._tiles_generator
+        )
+
```

Comment:
this property should be places after the `_tiles_generator` one because this way is not alphabetically ordered. Sorry